strings.md

```diff
@@ -3100,6 +3100,6 @@ The `wrap` method wraps the given string with an additional string or pair of st
 
     // "Laravel"
 
-    Str::is('is')->wrap(before: 'This ', after: ' Laravel!');
+    Str::of('is')->wrap(before: 'This ', after: ' Laravel!');
```

Comment:
Is it possible that you messed up your branch in some way so that your PR contains changes from other PRs that shouldn't be part of this one? 🤔

Reply:
I'm pretty sure that I'm not changing anything beside adding the Code Fencing. You can check my commit @shaedrich

Reply:
Yeah, 3f7600d is your commit, but e0bfeee..f65a8dc are not (see [commits tab](https://github.com/laravel/docs/pull/10196/commits))